### ISSUE TITLE
training epoch

### ISSUE THREAD

**Issue by Yiignorant** [whiteyunjie/ROAM#2]
Hello author, I'd like to ask generally how long one training epoch takes for the ROAM model?

**Comment by whiteyunjie** [comment]
Here is a reference data: one of our training sets has about 560 slides, and when the batch size is set to 8, it takes about 3 minutes to train an epoch using the ROAM model on an RTX A6000. The specific training time is related to the resolution of the slide and GPU type.

**Comment by Yiignorant** [comment]
Dear author,
Thank you for your reply. Since I cannot access the private dataset in your project, it should be reasonable for me to train the model using only the TCGA glioma dataset, right?







------------------ Original ------------------
From: Whiteyunjie ***@***.***&gt;
Date: Mon,Jun 30,2025 2:15 AM
To: whiteyunjie/ROAM ***@***.***&gt;
Cc: Ignorant_kai ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [whiteyunjie/ROAM] training epoch (Issue #2)



whiteyunjie left a comment (whiteyunjie/ROAM#2)
 
Here is a reference data: one of our training sets has about 560 slides, and when the batch size is set to 8, it takes about 3 minutes to train an epoch using the ROAM model on an RTX A6000. The specific training time is related to the resolution of the slide and GPU type.

**Comment by whiteyunjie** [comment]
@Yiignorant  Yes, thank you for your understanding. You may train and test the model using the TCGA glioma dataset. The corresponding label information is provided at https://doi.org/10.5281/zenodo.11469546.
